docs/user-guide/swarm-mode.md

```diff
@@ -119,14 +120,22 @@ docker-machine ssh manager "docker service create \
 	--name whoami0 \
 	--label traefik.port=80 \
 	--network traefik-net \
+	--label traefik.frontend.rule=Host:whoami0.traefik \
+	--label traefik.backend=whoami0 \
 	emilevauge/whoami"
+
 docker-machine ssh manager "docker service create \
 	--name whoami1 \
 	--label traefik.port=80 \
 	--network traefik-net \
+	--label traefik.frontend.rule=Host:whoam1.traefik \
+	--label traefik.backend=whoami1 \
+	--label traefik.backend.loadbalancer.sticky=true \
```

Comment:
Is this up-to-date ? (isn't it supposed to be `traefik.backend.loadbalance.swarm` ?)